cypress/integration/login.js

```diff
@@ -23,6 +23,80 @@ describe('Sourcebooks login', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Nadežda Žarikova');
         cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.main-nav__link--active').contains('Time Logging').should('have.css' , 'color' , 'rgb(64, 76, 237)');
+        cy.get('.calendar--today').contains(new Date().getDate());
+    })
+    it('Should be able to login with role Team Lead', function () {
+
+        cy.visit('/');
+        cy.get('[id="loginForm.userId"]').click({force:true});
+        cy.get('[aria-label="Nadežda Žarikova"]').click();
+        cy.get('[id="loginForm.role"]').click({force:true});
+        cy.get('[aria-label="Team Lead"]').click();
+        cy.get('[type="submit"]').click();
+
+        cy.url().should('include', '/time-logging');
+        cy.get('.page__title').contains('Timesheets')
+        cy.get('.calendar').should('be.visible')
+        cy.get('.tile.form').should('be.visible')
+        cy.get('.user-info__title').contains('Nadežda Žarikova');
+        cy.get('.main-nav').find('li').should('have.length', 2);
+        cy.get('.main-nav__link--active').contains('Time Logging').should('have.css' , 'color' , 'rgb(64, 76, 237)');
+        cy.get('.calendar--today').contains(new Date().getDate());
+    })
+

```

Comment:
Good job! However, to keep consistent either remove this empty line or add them between other test cases as well :)